hack/ci/prom_metric_linter.sh

```diff
@@ -49,7 +43,7 @@ while [[ $# -gt 0 ]]; do
 done
 
 # Get the metrics list
-go build -o _out/prom-metrics-collector "$METRICS_COLLECTOR_PATH/..."
+./hack/build/bazel-docker.sh go build -o _out/prom-metrics-collector /root/go/src/kubevirt.io/containerized-data-importer/tools/prom-metrics-collector
```

Comment:
@avlitman Hey, would appreciate an ack on this commit - https://github.com/kubevirt/containerized-data-importer/pull/3250/commits/db308632d4bb92e3730a1ee028275cc5902d4fee

Reply:
looks nice thanks! 
Just to be sure did you change a metric name to not start with `kubevirt_cdi` for example, then run `make lint-metrics` and see it fail/pass as expected?